### PR TITLE
FroggerGame task

### PR DESCRIPTION
**Pull request by OlexiyDobroskok**
### OO JS (Frogger)

[Demo](https://olexiydobroskok.github.io/Frogger__Game/) |
[Code base](https://github.com/OlexiyDobroskok/Frogger__Game)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+skinsList.prepend(...skinsListCreator());
+
+let score = 0;
+
+const result = document.createElement("p");
+result.classList.add("show__result");
+skinsList.after(result);
+result.innerHTML = `Score: ${score}`;
+
+function skinsListCreator() {
```
Comment (artem-trubin) [inline submissions/OlexiyDobroskok/ClassikFroggerGame/app.js:48]: We usually name functions after some action, so it must be a verb at its core. Like `createSkinsList()`.

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    skinImg.id = id;
+    listItem.prepend(skinImg);
+    return listItem;
+  });
+}
+
+let playerSprite = "images/char-cat-girl.png";
+
+skinsList.addEventListener("click", function ({ target }) {
+  skins.forEach((skin) => {
```
Comment (artem-trubin) [inline submissions/OlexiyDobroskok/ClassikFroggerGame/app.js:66]: This is better done with `Array.find()`.

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+Player.prototype.update = function () {};
+
+Player.prototype.reset = function () {
+  this.x = PLAYER_START_POSITION_X;
+  this.y = PLAYER_START_POSITION_Y;
+  increaseScore();
+};
+
+Player.prototype.render = function () {
+  ctx.drawImage(Resources.get(playerSprite), this.x, this.y);
```
Comment (artem-trubin) [inline submissions/OlexiyDobroskok/ClassikFroggerGame/app.js:130]: I believe `playerSprite` should be a part of the Player. Same way as Enemy's sprite.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Cool work! Let's make some changes, check the comments below.

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js (hunk 1 above)
Comment (OlexiyDobroskok) [reply submissions/OlexiyDobroskok/ClassikFroggerGame/app.js]: Done)

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js (hunk 2 above)
Comment (OlexiyDobroskok) [reply submissions/OlexiyDobroskok/ClassikFroggerGame/app.js]: Good advice,Thank you!)

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js (hunk 3 above)
Comment (OlexiyDobroskok) [reply submissions/OlexiyDobroskok/ClassikFroggerGame/app.js]: done

File: submissions/OlexiyDobroskok/ClassikFroggerGame/app.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+};
+
+const player = new Player(PLAYER_START_POSITION_X, PLAYER_START_POSITION_Y);
+
+const allEnemies = ENEMIES_START_POSITION_Y.map((yPosition) => {
+  const xPosition = Math.floor(Math.random() * -200);
+  return (enemy = new Enemy(xPosition, yPosition, ENEMY_START_SPEED));
```
[the remark this answers is not in the document]
Comment (OlexiyDobroskok) [reply submissions/OlexiyDobroskok/ClassikFroggerGame/app.js]: Oh....I am a noob< done :))

**Review by artem-trubin — APPROVED** [review]
Well done!

**Pull request merged**